jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -198,61 +200,60 @@ As stated earlier, all component Specifications which plan a Major or Minor rele
 
 ### Waves
 
-We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications.
+We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications. (updated specifications marked with asterisks)
 
 #### Wave 1
 
-- Jakarta Annotations
+- Jakarta Annotations*
 
 #### Wave 2
 
 - Jakarta Expression Language
-- Jakarta Interceptors
+- Jakarta Interceptors*
 - Jakarta Lang Model (may be released with Jakarta CDI [Core] in wave 3)
 
 #### Wave 3
 
-- Jakarta Activation
-- Jakarta Contexts and Dependency Injection
+- Jakarta Activation*
```

Comment:
Activation will just have a service release which might need to be noted here with text, like `Jakarta Activation* (service release only)` to avoid confusing people.

Reply:
The other service release only specs should be noted as well IMO.

Reply:
We do note the following service releases:
```
Jakarta Activation 2.1.2* service release
Jakarta JSON Processing 2.1.2* service release
Jakarta Mail 2.1.2* service release
```

Activation is part of Wave 3 and Mail is part of Wave 4 but Jakarta JSON Processing is not part of a wave (perhaps it should be part of Wave 1 like it was for EE 10).